<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1384</width>
    <height>791</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>dxfviewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonDxfParse">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>460</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>解析dxf文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSelFile">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择dxf文件...</string>
    </property>
   </widget>
   <widget class="GraphicsFrame" name="m_graphicsFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1181</width>
      <height>711</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMoveLeft">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>530</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMoveRight">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>530</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMoveDown">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>560</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMoveUp">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>500</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonZoomOut">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>610</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>放大</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonZoomIn">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>650</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>缩小</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetEntities">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>50</y>
      <width>161</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1384</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsFrame</class>
   <extends>QFrame</extends>
   <header>graphicsframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
